.github/scripts/cibw_before_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -ev
+
+# FIXME: this is currently only running for python 3.9 wheels
+if [[ $RUNNER_OS == "Windows" ]]; then
+    sdk_url=https://data.kitware.com/api/v1/file/6294ed244acac99f42543957/download
+elif [[ $RUNNER_OS == "macOS" ]]; then
+    sdk_url=https://data.kitware.com/api/v1/file/6294ed374acac99f4254396d/download
+elif [[ $RUNNER_OS == "Linux" ]]; then
+    # Need to install wget
+    yum install wget -y
+    sdk_url=https://data.kitware.com/api/v1/file/6294ed4c4acac99f425439a8/download
+fi
```

Comment:
These are currently hard-coded to some examples on `data.kitware.com`. We will need to find a way to point these all to their respective python version and operating system on `vtk.org` when we start getting weekly wheel SDKs uploaded.

Reply:
These are now downloaded automatically in the `setup.py` for the particular platform and python version.